Create login.ivdnt.org.xml

rvanvliet opened the pull request.

rvanvliet committed:
Create login.ivdnt.org.xml

andmor- committed:
Merge branch 'master' into master

andmor- committed:
Add mdattr namespace to login.ivdnt.org.xml

rvanvliet commented:
@andmor- thank you for the help with modifying the xml file.

andmor- commented:
No problem. If you want we can already push this forward for testing but only with the CLARIN IdP. For propagation to other IdPs (and for production status) you will need some extra changes.
Form the top of my mind you will need a `<mdui:UIInfo>` section and two other contacts: "administrative" and "support"
Check the files for the other SPs for examples. Like this one: https://github.com/clarin-eric/SPF-SPs-metadata/blob/master/metadata/sp.catalog.clarin.eu.xml

rvanvliet committed:
Added more contacts and added UIInfo to login.ivdnt.org.xml

rvanvliet commented:
> No problem. If you want we can already push this forward for testing but only with the CLARIN IdP. For propagation to other IdPs (and for production status) you will need some extra changes. Form the top of my mind you will need a `<mdui:UIInfo>` section and two other contacts: "administrative" and "support" Check the files for the other SPs for examples. Like this one: https://github.com/clarin-eric/SPF-SPs-metadata/blob/master/metadata/sp.catalog.clarin.eu.xml

Thank you! I've added extended the contacts and added the mdui:UIInfo section, is this valid?

rvanvliet committed:
added xmlns

andmor- committed:
Merge branch 'master' into master
